### PR TITLE
Added posts

### LINKED ISSUE #22: Create Posts view (subpage)
- available to everyone (including not logged in users)
- for now - empty subpage
- add button to the navbar

### PR DESCRIPTION
**Pull request by wdoganowski**
Dodałem posty, zainstalowałem je i dodałem tymczasowo do strony głównej
Wyświetlane są tylko posty z przeszłymi datami do publikacji (pub_date_time). Dla superuser jest na navarze skrót do przeglądania i edycji wszystkich postów

### CODE REVIEW

File: harc_game_web/apps/posts/admin.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Post
```
Comment (zkunik) [inline harc_game_web/apps/posts/admin.py]: it is preferred to specify full path in import, in this case `apps.posts.models`

File: harc_game_web/apps/posts/models.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		Treść
+		Autor
+		Link do filmiku/obrazu
+    """
+    
+    slug = models.SlugField(max_length=100, unique=True)
+    date_time = models.DateTimeField('Data utworzenia')
+    pub_date_time = models.DateTimeField('Kiedy ma być opublikowany')
+    title = models.CharField(max_length=200)
+    content = models.CharField(max_length=2000)
```
Comment (zkunik) [inline harc_game_web/apps/posts/models.py]: maybe we should use `TextField`?

File: harc_game_web/apps/posts/views.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+from django.http import HttpResponseRedirect
+from django.utils import timezone
+from django.db.models import F
+
+class PostForm(forms.ModelForm):
+    class Meta:
+        model = Post
+        fields = ['pub_date_time', 'title', 'content', 'link1', 'link2', 'link3']
+  
+def ListActivePosts(request):
```
Comment (zkunik) [inline harc_game_web/apps/posts/views.py]: following PEP8, names of functions and methods should be lowercase with words separated by underscore (e.g. `list_active_posts`. Please fix it here and in other places 🙏

File: harc_game_web/apps/posts/views.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+	return render(request, 'posts/list.html', {'posts': posts})
+
+def ViewPost(request, slug):
+	"""
+	Function to list a single post
+	"""
+	post = Post.objects.get(slug=slug)
+	return render(request, 'posts/view.html', {'post': post})
+	
+def NewPost(request):
```
Comment (zkunik) [inline harc_game_web/apps/posts/views.py]: This and other functions (edit, delete post) should be available only for superuser. Decorator `@staff_member_required` or other check should be used

File: .gitignore [hunk 5]
```diff
@@ -6,3 +6,4 @@ harc_game_web/db.sqlite3
 **/migrations/*
 !**/migrations/__init__.py
 media/
+.DS_Store
```
Comment (zkunik) [inline .gitignore:9]: this file is specified here, however it is still added to the PR ;)

**Review by zkunik — CHANGES_REQUESTED** [review]
Great job! I'm happy that we have these posts implemented 🎉 Please fix minor issues and we are ready to go!

File: harc_game_web/apps/posts/views.py (hunk 3 above)
Comment (wdoganowski) [reply harc_game_web/apps/posts/views.py:13]: Ok... I saw you used mixed mode and did not know what is preference. Will do

File: harc_game_web/apps/posts/models.py (hunk 2 above)
Comment (wdoganowski) [reply harc_game_web/apps/posts/models.py]: https://django.programmingpedia.net/en/knowledge-base/7354588/django-charfield-vs-textfield

I'll remove the chars limit from content

File: harc_game_web/apps/posts/views.py (hunk 3 above)
Comment (wdoganowski) [reply harc_game_web/apps/posts/views.py:13]: Or maybe not...

File: harc_game_web/apps/posts/views.py (hunk 4 above)
Comment (wdoganowski) [reply harc_game_web/apps/posts/views.py]: It is actually done in html templat. Will try @staff_member_required

**Comment by wdoganowski** [comment]
Myślę, że wszystko poprawiłem...

**Review by zkunik — CHANGES_REQUESTED** [review]
file `.DS_store` still added to the PR

**Review by zkunik — APPROVED** [review]

**Pull request merged**